index.html

```diff
@@ -0,0 +1,263 @@
+                    <div class="speaker-info">
+                        <h5 class="speaker-name" id="speak-name3"></h5>
+                        <p class="speaker-qualification main-color" id="speak-quali3">Risk analyst at International Fund for Agricultural Development</p>
+                        <hr class="speaker-hr">
+                        <p class="small-text" id="speak-info3">Omotola studies risks of investing in agricultural firms located in desert regions</p>
+                    </div>
+                   
+                </div>
+                <div class="speaker-card">
+                    <img class="speaker-image" id="speak-img4" src="" alt="speaker-picture">
+                    <div class="speaker-info">
+                        <h5 class="speaker-name" id="speak-name4"></h5>
+                        <p class="speaker-qualification main-color" id="speak-quali4">Risk analyst at International Fund for Agricultural Development</p>
+                        <hr class="speaker-hr">
+                        <p class="small-text" id="speak-info4">Omotola studies risks of investing in agricultural firms located in desert regions</p>
+                    </div>
+                   
+                </div>
+                <div class="speaker-card">
+                    <img class="speaker-image" id="speak-img5" src="" alt="speaker-picture">
+                    <div class="speaker-info">
+                        <h5 class="speaker-name" id="speak-name5"></h5>
+                        <p class="speaker-qualification main-color" id="speak-quali5">Risk analyst at International Fund for Agricultural Development</p>
+                        <hr class="speaker-hr">
+                        <p class="small-text" id="speak-info5">Omotola studies risks of investing in agricultural firms located in desert regions</p>
+                    </div>
+                   
+                </div>
+                <div class="speaker-card">
+                    <img class="speaker-image" id="speak-img6" src="" alt="speaker-picture">
+                    <div class="speaker-info">
+                        <h5 class="speaker-name" id="speak-name6"></h5>
+                        <p class="speaker-qualification main-color" id="speak-quali6">Risk analyst at International Fund for Agricultural Development</p>
+                        <hr class="speaker-hr">
+                        <p class="small-text" id="speak-info6">Omotola studies risks of investing in agricultural firms located in desert regions</p>
+                    </div>
+                   
+                </div>
+            </div>
+           
+            <div class="more-speakers">
+                <a class="show-more"  href="#"></a><button id="show-more" class="more-speakers-button">More <img class="angle-down" src="images/angle-down.png" alt="angle-down-icon"></button></a>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] Please all this section is supposed to be generated dynamically in a js file, kindly delete this part in the HTML file, you should have only the `<section>` tag with class in all this part 🙃 🙃